src/components/draft-js-editor/DraftJSEditor.stories.js

```diff
@@ -1,8 +1,7 @@
 // @flow
+/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
added this to avoid having to capitalize all stories to and keeping this as camelcase